plugins/isecl/README.md

```diff
@@ -0,0 +1,99 @@
+# iSecL 
+
+## Overview
+
+[Intel® SecL-DC](https://intel-secl.github.io/docs/4.2/) is an open-source
+remote attestation implementation comprising of a set of building blocks
+that utilize Intel Security features to discover, attest, and enable critical
+foundation security and confidential computing use-cases. It applies the 
+remote attestation fundamentals and standard specifications to maintain a 
+platform data collection service and an efficient verification engine to
+perform comprehensive trust evaluations.
+
+This plugin uses Intel(R) SecL Key Broker Service to securely transfer CA
+private keys to the [trusted-certificate-issuer](https://github.com/intel/trusted-certificate-issuer)
+SGX Enclave from a KMIP-compliant key management server.
+
+## Setup
+
+### Install iSecL components
+
+Below are the minimum required Intel(R) SecL-DC components for using this plugin:
+
+* Certificate Management Server ([CMS](https://github.com/intel-secl/intel-secl/tree/master/pkg/cms))
+* Authentication and Authorization Service ([AAS](https://github.com/intel-secl/intel-secl/tree/master/pkg/aas))
+* Key Broker Service ([KBS](https://github.com/intel-secl/tree/master/pkg/kbs))
+* SGX Caching Service ([SCS](https://github.com/intel-secl/sgx-caching-service))
+* SGX Quote Verification Service ([SQVS](https://github.com/intel-secl/sgx-verification-service))
+
+Other dependencies:
+* [Postgres SQL](https://pypi.org/project/pgdb/)
+* A [KMIP](https://en.wikipedia.org/wiki/Key_Management_Interoperability_Protocol)-compliant
+key management service, such as [PyKMIP](https://pykmip.readthedocs.io/en/latest/server.html) or,
+[HashiCorp Vault](https://www.vaultproject.io/docs/secrets/kmip)
+
+You can use [setup_isecl.sh](./tools/setup-isecl.sh) to setup the above components.
+
+[Intel(R) SecL utils](https://github.com/intel-secl/utils) repo provides helper
+scripts to deploy on different environments.
+
+### Configure the client TLS certificates
+
+```sh
+export TAC_USER="tacuser" # TAC client service username, defined in setup-isecl.sh
+export TAC_PASSWORD="tacpassword" #TAC client service password, defined in setup-isecl.sh
+export TAC_TOKEN=$(curl --noproxy "*" -k -X POST https://avalluriGigabyte.fi.intel.com:8444/aas/v1/token -d '{"username": "'$TAC_USER'", "password": "'$TAC_PASSWORD'"}')
```

Comment:
Maybe remove the avalluriGigabyte.fi.intel.com address so that no-one accidentally uses it.

Reply:
Done.